src/serde_tokenstream.rs

```diff
@@ -57,7 +57,78 @@ pub fn from_tokenstream<'a, T>(tokens: &'a TokenStream) -> Result<T>
 where
     T: Deserialize<'a>,
 {
-    let mut deserializer = TokenDe::from_tokenstream(tokens);
+    let deserializer = TokenDe::from_tokenstream(None, tokens);
+    from_tokenstream_impl(deserializer)
+}
+
+/// Deserialize an instance of type T from a token stream with data inside,
```

Comment:
```suggestion
/// Deserialize an instance of type T from a TokenStream with data inside,
```

For consistency with `from_tokenstream`?